cloud-sql/mysql/Startup.cs

```diff
@@ -112,13 +112,13 @@ PRIMARY KEY (vote_id)
         public static MySqlConnectionStringBuilder GetMySqlConnectionString()
         {
             MySqlConnectionStringBuilder connectionString; 
-            if (Environment.GetEnvironmentVariable("DB_HOST") != null)
+            if (Environment.GetEnvironmentVariable("INSTANCE_HOST") != null)
             {
-                connectionString = NewMysqlTCPConnectionString();
+                connectionString = MySqlTcp.NewMysqlTCPConnectionString();

```

Comment:
Is `return` idiomatic here instead of if/else?

Reply:
Defer to amanda-tarafa@ or jskeet@ on this one.

Reply:
Since we are using `connectionString` to set some properties after it was initialized (see lines >= 127) we need to do `if...else` here.

If this method was inmediately returning after initializing `connectionString` then the most idiomatic thing to do is probably:

```chsarp
return Environment.GetEnvironmentVariable("INSTANCE_HOST") is null ? MySqlUnix. ... : MySqlTcp. ...;
```